nifi-nar-bundles/nifi-media-bundle/nifi-media-processors/src/test/java/org/apache/nifi/processors/image/ExtractImageMetadataTest.java

```diff
@@ -37,7 +37,7 @@
     private static String BMP_HEADER = "BMP Header.";
     private static String JPEG_HEADER = "JPEG.";
     private static String GIF_HEADER = "GIF Header.";
-    private static String PNG_HEADER = "PNG.";
+    private static String PNG_HEADER = "PNG-";
```

Comment:
If by "bridge the header/attribute" you mean adjust keys/values to match the old version, I don't have a list identifying what changed.  This particular change was caught on chance because of a unit test.

I'm open to suggestions, but the only option I see that has doesn't break backward compatiblity is isolate the processors back into separate NARs.